lib/mayaUsd/render/MaterialXGenOgsXml/libraries/mx_srgb_texture_to_aces_2065_1_color3.glsl

```diff
@@ -1,8 +1,8 @@
-#include "libraries/stdlib/genglsl/lib/mx_transform_color.glsl"
+#include "libraries/adsk/maya/genglsl/lib/usd_transform_color.glsl"
```

Comment:
Changes requires copying the old mx_transform_color.glsl to MayaUSD since that file no longer exists in 1.38.8.